README.md

```diff
@@ -14,13 +14,25 @@ in lightweight virtual machines, called microVMs, which combine the security and
 isolation properties provided by hardware virtualization technology with the
 speed and flexibility of containers.
 
+## Supported platforms
+
+Firecracker officially supports the following CPUs for production usecases:
```

Comment:
```suggestion
We continuously test Firecracker on machines with the following CPUs micro-architectures:
```

I think we should set specific expectations for what we test on, but that should be distinct from what we support (see comment below).